### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by sonninnos**
Statusbar indicator for normal fps and warp mode cleaned up.

One less core option with a combined drive sound emulation option.

NumLock as drive LED indicator option only for WIN32 for now, because that I can test, so assistance is much appreciated!

### CODE REVIEW

File: libretro/libretro-core.c [hunk 1]
```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__
+#include <windows.h>
```
Comment (rsn8887) [inline libretro/libretro-core.c]: Are you adding external dependencies here? Are we sure the buildbot has these installed? Were these functions X* used before anywhere else in this core or in any other Win32 core?

This needs checking I think to make sure the buildbot can compile this stuff.

File: libretro/libretro-core.c [hunk 2]
```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__
+#include <windows.h>
+#else 
+#include <X11/Xlib.h>
```
Comment (rsn8887) [inline libretro/libretro-core.c]: See above.

File: libretro/libretro-core.c [hunk 3]
```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__
```
Comment (rsn8887) [inline libretro/libretro-core.c]: Why is there a dual Win32 check here?

File: libretro/libretro-core.c (hunk 2 above)
Comment (rsn8887) [reply libretro/libretro-core.c]: Hmm you cannot assume x11 is available anywhere. Also it is inside a Win32 guard.

**Comment by rsn8887** [comment]
I don’t think we want the external dependency on windows.h functions here and on X11 functions anytime in the future, as far as I understand. 

I will not merge until I have a clear consensus from the other Retroarch devs.

File: libretro/libretro-core.c (hunk 3 above)
Comment (rsn8887) [reply libretro/libretro-core.c]: It seems this would break on all non-Win32 non-X11 platforms were the dual check removed.

**Comment by rsn8887** [comment]
Can you remove the led stuff? There’s a pretty clear no on the win32 stuff and a definite no on he X11 stuff from bparker.

The led thing has to be implemented in libretro-common in a platform agnostic way before we can implement it here.

**Comment by sonninnos** [comment]
Oh well, it is what it is.. LED shenanigans removed.

File: libretro/libretro-core.c (hunk 3 above)
Comment (sonninnos) [reply libretro/libretro-core.c]: It was just as a placeholder for future additions

**Comment by rsn8887** [comment]
Ok thanks for all the hard work :)

**Pull request merged**